client/src/context/reducer.js

```diff
@@ -0,0 +1,52 @@
+let user = localStorage.getItem("user")
+  ? JSON.parse(localStorage.getItem("user"))
+  : "";
+let token = localStorage.getItem("token");
+let isAuthenticated = localStorage.getItem("isAuthenticated") ? true : false;
+
+export const initialState = {
+  user: "" || user,
+  token: "" || token,
```

Comment:
if the above condition is false, token is by default to an empty string, we can safely set this to only token.